handlers/update.go

```diff
@@ -43,7 +42,6 @@ func MakeUpdateHandler(functionNamespace string, clientset *kubernetes.Clientset
 
 		if len(deployment.Spec.Template.Spec.Containers) > 0 {
 			deployment.Spec.Template.Spec.Containers[0].Image = request.Image
-			deployment.Spec.Template.Spec.Containers[0].ImagePullPolicy = v1.PullAlways
```

Comment:
I'd rather comment this line out and state why it is left that way than remove it without the context.

Reply:
I'm not a big fan of comments on deleted code - I could get more specific in the commit message.
Perhaps more importantly on this one I was thinking more - while the apispec claims it's not updatable, that's at the pod level; so in a deployment template, it probably is updatable since that creates a new replicaset.  It's worth deciding on-purpose however if we want it to be updatable in faasnetes, since it's effectively a global.  It could lead to unexpected behavior; consider e.g. in-place deployed/working functions that get mutated to `IfNotPresent` or `Never` (inadvertently or otherwise) after a faasnetes redeployment, which would break them.
I'm 50/50 on this one, thoughts?